content/applications/general/iot/config/updating_iot.rst

```diff
@@ -73,6 +73,70 @@ Lastly, click :guilabel:`Flash`, and wait for the process to finish.
    An alternative software for flashing the micro SD card is `Raspberry Pi Imager
    <https://www.raspberrypi.com/software/>`_.
 
+Windows IoT update
+==================
+
+Windows virtual :abbr:`IoT (Internet of Things)` box may occasionally need an update to work
+properly. The following processes are the un-installation and re-installation of the Windows virtual
+:abbr:`IoT (Internet of Things)` box.
+
+.. important::
+   Before un-installation of a previous version of the Windows virtual :abbr:`IoT (Internet of
+   Things)` box, check `Odoo's download page <https://odoo.com/download>`_ to ensure a newer version
```

Comment:
true, but how ? this page does not display anywhere the version that the client will download, it only shows a "Download" button. It might confuse customers making them think they have to download "later versions" like version 17 instead of 16.

Maybe it would be a good idea to inform on what really download when you click the button (does it update the file daily ? weekly ?).

Also note that there is this other page:
https://nightly.odoo.com/
which contains the latest release per platform, for instance:
https://nightly.odoo.com/17.0/nightly/windows/
where you can acautlly see the real executables difference (which will each contains the version of the code of odoo that was there on this day)